### PR TITLE
Cursor tweaks

### PR DESCRIPTION
**Pull request by epicabsol**
Added unique cursors for the dither, stroke, and lighting tools, along with repositioning the hotspot of some of the cursors.

### CODE REVIEW

File: src/css/tools.css [hunk 1]
```diff
@@ -31,33 +31,39 @@
...
+  cursor: url(../img/cursors/pen.png) 1 14, pointer;
+}
+
 .tool-dithering .drawing-canvas-container:hover {
-  cursor: url(../img/cursors/pen.png) 0 15, pointer;
+  cursor: url(../img/cursors/dither.png) 1 14, pointer;
+}
+
+.tool-lighten .drawing-canvas-container:hover {
+  cursor: url(../img/cursors/lighting.png) 1 15, pointer;
```
Comment (juliandescottes) [inline src/css/tools.css]: Could you rename lighting.png to lighten.png to be consistent with the current tool name?

**Comment by juliandescottes** [comment]
Thanks for the PR! 
Have a look at my comment, will merge it right after.

**Comment by epicabsol** [comment]
Renamed lighting -> lighten.

**Comment by juliandescottes** [comment]
Thanks for updating the name, merging.

**Pull request merged**